qcodes/instrument_drivers/AlazarTech/ats_api.py

```diff
@@ -0,0 +1,502 @@
+"""
+This module provides a class that encapsulates Alazar ATS API, 
+:class:`AlazarATSAPI`. The class is used to expose Alazar API functions
+of its C library in a python-friendly way.
+"""
+
+
+from typing import Dict, Tuple, Union, Any
```

Comment:
Verified that all signatures in this file matches the latest sdk manual

Reply:
oh, thanks a lot!